photutils/psf/tests/test_photometry.py

```diff
@@ -0,0 +1,202 @@
+        sources['x_mean'] = [13, 18, 25]
+        sources['y_mean'] = [16, 16, 25]
+        sources['x_stddev'] = [sigma_psf, sigma_psf, sigma_psf]
+        sources['y_stddev'] = sources['x_stddev']
+        sources['theta'] = [0, 0, 0]
+        sources['id'] = [1, 2, 3]
+        sources['group_id'] = [1, 1, 2]
+        tshape = (32, 32)
+        
+        # generate image with read-out noise (Gaussian) and
+        # background noise (Poisson)
+        image = (make_gaussian_sources(tshape, sources) +
+                 make_noise_image(tshape, type='poisson', mean=6.,
+                                  random_state=1) +
+                 make_noise_image(tshape, type='gaussian', mean=0.,
+                                  stddev=2., random_state=1)) 
+
+        bkgrms = StdBackgroundRMS(sigma=3.)
+        std = bkgrms(image)
+
+        daofind = DAOStarFinder(threshold=5.0*std,
+                                fwhm=sigma_psf*gaussian_sigma_to_fwhm)
+        daogroup = DAOGroup(1.5*sigma_psf*gaussian_sigma_to_fwhm)
+        median_bkg = MedianBackground(sigma=3.)
+        psf_model = IntegratedGaussianPRF(sigma=sigma_psf)
+        fitter = LevMarLSQFitter()
+        photometry = DAOPhotPSFPhotometry(find=daofind, group=daogroup,
+                                          bkg=median_bkg, psf=psf_model,
+                                          fitter=LevMarLSQFitter(),
+                                          niters=1, fitshape=(11,11))
+
+        result_tab, residual_image = photometry(image=image)
+
+        assert_allclose(result_tab['x_fit'], sources['x_mean'], rtol=1e-1)
+        assert_allclose(result_tab['y_fit'], sources['y_mean'], rtol=1e-1)
+        assert_allclose(result_tab['flux_fit'], sources['flux'], rtol=1e-1)
+        assert_array_equal(result_tab['id'], sources['id'])
+        assert_array_equal(result_tab['group_id'], sources['group_id'])
+        assert_allclose(np.mean(residual_image), 0.0, atol=1e1)
+
+        # test fixed photometry
+        psf_model.x_0.fixed = True
+        psf_model.y_0.fixed = True
+        photometry = DAOPhotPSFPhotometry(group=daogroup, bkg=median_bkg,
+                                          psf=psf_model, fitter=LevMarLSQFitter(),
+                                          fitshape=(11,11))
+
+        pos = Table(names=['x_0', 'y_0'], data=[sources['x_mean'],
+                                                sources['y_mean']])
+        result_tab, residual_image = photometry(image=image, positions=pos)
+
+        assert_array_equal(result_tab['x_fit'], sources['x_mean'])
+        assert_array_equal(result_tab['y_fit'], sources['y_mean'])
+        assert_allclose(result_tab['flux_fit'], sources['flux'], rtol=1e-1)
+        assert_array_equal(result_tab['id'], sources['id'])
+        assert_array_equal(result_tab['group_id'], sources['group_id'])
+        assert_allclose(np.mean(residual_image), 0.0, atol=1e1)
+
+
+    def test_complete_photometry_two(self):
+        sigma_psf = 2.0
```

Comment:
short sentence what's different from test one. Just if this fails and somebody needs to understand what's going on five years from now (or, looking at how long IRAF is already used in astronomy, 30 years from now when we try to run this on Python 7.1)